src/VscPreviewExtension/package.json

```diff
@@ -70,13 +70,16 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "node ./node_modules/vscode/bin/compile",
-    "compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
+    "vscode:prepublish": "tsc -p ./",
+    "compile": "tsc -watch -p ./",
```

Comment:
do we need depend on global command `tsc`? how about using `node ./node_modules/typescript/bin/tsc` directly